packages/design-system/src/components/analytics/SendAnalytics.test.ts

```diff
@@ -1,6 +1,6 @@
-import { sendAnalyticsEvent } from './SendAnalytics';
+import { sendLinkEvent } from './sendAnalytics';
```

Comment:
[fix] the casing on the file name is causing unit tests to fail

Reply:
🤦‍♂️ I really thought TS would catch these for me.